app/collections/collaborators.cjsx

```diff
@@ -150,11 +150,11 @@ RoleRow = React.createClass
       @props.roleSet.delete()
 
     promise
-      .catch (error) =>
-        @setState { error }
       .then =>
         callback =>
-          @setState saving: false
+        @setState saving: false
```

Comment:
The problem here is that `@setState saving: false` wasn't triggering when a role was checked on (line 140) because the default callback was being used and skipping the state change. Changing the indention solved this, but perhaps there is a better option?